database/seed.js

```diff
@@ -33,60 +33,63 @@ const booking_schema = new mongoose.Schema({
   weeknight_discount: Number,
   instant_book: Boolean,
   request_to_book: Boolean,
-  addons_names: [String],
-  addons_price: Number,
   subTotal: Number
 });
 
 const Booking = mongoose.model('Booking', booking_schema);
 
 const seedDb = async () => {
-
   for (let i = 0; i <= 99; i++) {
-
     monthMaker = function() {
       let months = [5, 6, 7, 8];
       let month = months[Math.floor(Math.random() * months.length)];
       return month;
     };
-
     monthDays = function(month) {
       let days = [];
-      if (month !== 6) {
-        days = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30];
-      } else {
+      if (month === 2) {
+        days = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28];
```

Comment:
there is a shortcut way you can populate an array with consecutive numbers. 
`Array.from({length: 28}, (_, i) => i + 1);`

Reply:
Thank you fixed doc at index 0 and thx for the short-hand way to generate arrays